index.d.ts

```diff
@@ -1,18 +1,28 @@
 declare module 'riteway' {
   export function describe(label: string, callback: describeCallback): void
-  
+
   export function Try<U extends any[], V>(fn: (...args: U) => V, ...args: U): any
 
-  type describeCallback = (should: should) => Promise<void>
+  export function createStream(opts: CreateStreamOptions): ReadableStream
+
+  type end = () => void
 
-  type should = (label?: string) => { assert: assert }
+  interface assert {
+    (assertion: Assertion): void
+    end: end
+    assert: assert
+  }
 
-  type assert = (assertion: Assertion) => void
+  type describeCallback = (assert: assert) => Promise<void>
 
   interface Assertion {
-    given: string
-    should?: string
-    actual: any
-    expected: any
+    readonly given: any
+    readonly should?: string
```

Comment:
`should` should no longer be optional as it no longer gets a default from the old should function.

Reply:
Oops!